pkg/apis/kops/v1alpha2/networking.go

```diff
@@ -256,6 +256,8 @@ type AmazonVPCNetworkingSpec struct {
 	Env []EnvVar `json:"env,omitempty"`
 }
 
+const CiliumIpamEni = "eni"
```

Comment:
@olemarkus moving your comment from the broken PR https://github.com/kubernetes/kops/pull/11534#discussion_r642598335

Reply:
you suggest including the above to CiliumSpec if i understand correctly?

Reply:
I was thinking of something like 


```
type CiliumIpam string

const CiliumIpamENI CiliumIpam = "eni" 
```

Reply:
how about moving this into here to make things simpler? --> https://github.com/kubernetes/kops/blob/master/pkg/model/components/cilium.go

wdyt?

Reply:
tbh, i believe we can completely remove it

Reply:
ok, i made the changes as you suggested ^^ let's solve to a different PR if we need to revisit where this should be placed

Reply:
reverted, as this one will need additional changes, out of scope of PR i guess:
```
# k8s.io/kops/pkg/model/iam
pkg/model/iam/iam_builder.go:272:122: invalid operation: b.Cluster.Spec.Networking.Cilium.Ipam == kops.CiliumIpamEni (mismatched types string and kops.CiliumIpam)
pkg/model/iam/iam_builder.go:327:122: invalid operation: b.Cluster.Spec.Networking.Cilium.Ipam == kops.CiliumIpamEni (mismatched types string and kops.CiliumIpam)
# k8s.io/kops/pkg/model/iam
vet: pkg/model/iam/iam_builder.go:272:84: cannot compare b.Cluster.Spec.Networking.Cilium.Ipam == kops.CiliumIpamEni (mismatched types string and kops.CiliumIpam)
```